run-tests.php

```diff
@@ -25,10 +25,14 @@
 
 /* $Id$ */
 
+/* Temporary variables while this file is being refactored. */
+/** @var ?JUnit */
+$junit = null;
+
+/* End temporary variables. */
+
 /* Let there be no top-level code beyond this point:
  * Only functions and classes, thanks!
- *
- * Minimum required PHP version: 7.1.0
```

Comment:
This line should stay, possibly with a newer minimum version.

Reply:
By the way, I'm happy for the version to be bumped up to 7.4.

Reply:
Can return, but what's the use case for supporting anything but current version? Does this file ever gets run against an earlier PHP version?

Reply:
The PHP binary executing run-tests.php and the PHP binary being tested are *usually* the same, but don't have to be. Especially when testing a badly broken binary that is not capable of correctly running run-tests.php itself.

Reply:
Done.